content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/scrap_inventory.rst

```diff
@@ -34,42 +34,34 @@ scrapped from inventory. In the :guilabel:`Quantity` field, change the value to
 product that should be scrapped (by default, this value is set to `1.00`).
 
 .. image:: scrap_inventory/scrap-inventory-new-scrap-order.png
-   :align: center
    :alt: Filled out new scrap order form with product details.
 
-The :guilabel:`Source Location` defaults to the location where the product is currently stored. The
-:guilabel:`Scrap Location` defaults to the designated scrap location (:guilabel:`Virtual
-Locations/Scrap`). Either of these locations can be changed by selecting a different location from
-their respective drop-down menus.
+If the product is tracked by lot or serial number, specify the tracking number in the
+:guilabel:`Lot/Serial Numbers` field.
 
-If the scrapping is tied to a specific existing operation, specify the operation in the
-:guilabel:`Source Document` field.
+The :guilabel:`Source Location` defaults to the location where the product is currently stored. The
+:guilabel:`Scrap Location` defaults to the designated scrap location (:guilabel:`Inventory
+adjustment`). Either of these locations can be changed by selecting a different location from their
+respective drop-down menus.
```

Comment:
I'm not sure if it's necessary to clarify that these location options don't appear by default unless locations setting is activated from the "Inventory" configuration.